src/screens/Exchange/Exchange.js

```diff
@@ -533,6 +552,23 @@ class ExchangeScreen extends React.Component<Props, State> {
     });
   };
 
+  setFromAmount = amount => {
+    this.props.resetOffers();
+    this.setState({
```

Comment:
it's not safe to use this.state.value, I would consider converting into this way:
```
this.setState(prevState) => {
  return {
value: {
        ...prevState.value,
        fromInput: {
          ...prevState.value.fromInput,
          input: amount,
        },
      }};
}, () => { ...
```

Reply:
Thanks! 👍 But then if `prevState` is used then state updated callback is not acceptable, these values in state are always present so I think it's safe to spread state like this and pass callback as we need to validate the input right after state was updated. I will look for another option how to pass validation and perform search, but it can be we will need to leave it this way.

Reply:
Thanks, updated 👍